### PR TITLE
[FIX] Use new scaler to retain original unaltered scaler

### LINKED ISSUE #30: Scaler parameters overridden within MovingAveragePCA
@eurunuela in #26, it looks like you commented out code that generated a `Scaler` within the subsampling procedure, and instead reused the one for the general scaling:

https://github.com/ME-ICA/mapca/blob/6c1d58c0256660561a96c0183d4e477b4fd16c4e/mapca/mapca.py#L174-L177

My concern is that re-fitting `self.scaler_` based on new data will make it invalid for rescaling data _back_ to the original scale, as here:

https://github.com/ME-ICA/mapca/blob/6c1d58c0256660561a96c0183d4e477b4fd16c4e/mapca/mapca.py#L331-L332

Does that make sense, or was there a specific reason you got rid of the original approach? Apologies for not noticing this in #26 and bringing it up while that PR was open.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #30.

Changes proposed:

- When variance-normalizing subsampled data, use a temporary scaler instead of the object's primary scaler, because `fit_transform`ing a scaler overwrites the mean and std attributes that are necessary for rescaling the data in `inverse_transform`.

### CODE REVIEW

**Comment by notZaki** [comment]
An alternative could be to use [`scale(dat, axis = 2)`](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.scale.html) for one-off usage.

**Review by eurunuela — APPROVED** [review]
